common/src/test/java/org/apache/uniffle/common/util/ChecksumUtilsTest.java

```diff
@@ -21,18 +21,27 @@
 import java.io.FileOutputStream;
 import java.nio.ByteBuffer;
 import java.nio.channels.FileChannel;
-import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Random;
 import java.util.zip.CRC32;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class ChecksumUtilsTest {
 
+  @TempDir File tempDir;
+
+  @BeforeEach
+  void beforeEach() {
+    assertTrue(this.tempDir.isDirectory());
+  }
+
```

Comment:
This beforeEach is not necessary.

Reply:
Thank you for the feedback and I updated the part you mentioned.